### ISSUE TITLE
NSP vulnerability for protobufjs <=6.8.5

### ISSUE THREAD

**Issue by olastor** [firebase/firebase-js-sdk#813]
### [REQUIRED] Describe your environment

  * Firebase SDK version: 5.0.2
  * Firebase Product: firestore

#### Steps to reproduce:

`nsp check`

#### Relevant Code:

Didn't find this in the existing issues. NSP finds this vulnerability with latest version 5.0.2 installed. Can somebody reproduce?

```
(+) 1 vulnerability found
┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Denial of Service                                                  │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ protobufjs                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 5 (Medium)                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 5.0.2                                                              │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <=6.8.5                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ >=6.8.6                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ meditation-plus-angular@0.0.0 > firebase@5.0.2 >                   │
│            │ @firebase/firestore@0.5.2 > grpc@1.10.1 > protobufjs@5.0.2         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/605                             │
└────────────┴────────────────────────────────────────────────────────────────────┘

```

**Comment by omar10594** [comment]
the vulnerability was fixed on version 5.0.3 of protobufjs, so you just need to update to that version, for example with yarn -> `yarn upgrade firebase`

**Comment by olastor** [comment]
Thanks :+1:

**Comment by mboughaba** [comment]
Hi,

This is still an issue. Protobufjs 5.0.3 seems vulnerable too.

```shell
λ nsp check --reporter table
(+) 1 vulnerability found
┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Denial of Service                                                  │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ protobufjs                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 5 (Medium)                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 5.0.3                                                              │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <=6.8.5                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ >=6.8.6                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ alita-web@0.1.2 > firebase@5.0.4 > @firebase/firestore@0.5.4 >     │
│            │ grpc@1.11.3 > protobufjs@5.0.3                                     │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/605                             │
└────────────┴────────────────────────────────────────────────────────────────────┘
```

Cheers
